packages/amplify-provider-awscloudformation/src/aws-utils/aws-s3.ts

```diff
@@ -93,9 +94,9 @@ export class S3 {
   private attachBucketToParams(s3Params: $TSAny, envName?: string):$TSAny {
     // eslint-disable-next-line no-prototype-builtins
     if (!s3Params.hasOwnProperty('Bucket')) {
-      const projectDetails = this.context.amplify.getProjectDetails();
       if (!envName) envName = this.context.amplify.getEnvInfo().envName;
-      const projectBucket = projectDetails.teamProviderInfo[envName][providerName].DeploymentBucketName;
+      const teamProviderInfo = stateManager.getTeamProviderInfo();
```

Comment:
same here: Should this be updated to use the new parameter manager?

Reply:
same thing. provider block is not migrated yet